github/codeowners.py

```diff
@@ -116,38 +118,146 @@ def resolve_team_members(
     if not team_or_none:
         logger.warning('failed to lookup team {t}'.format(t=team_name))
         return []
-    for member in map(github_api.user, team_or_none.members()):
-        if member.email:
-            yield member.email
+    yield from map(github_api.user, team_or_none.members())
```

Comment:
nit: rather use generator-comprehension (`return (github_api.user(member) for member in team_or_non.members()`)